packages/editor-ui/src/components/MainSidebar.vue

```diff
@@ -15,7 +15,7 @@
 
 				<el-menu-item index="logo" class="logo-item">
 					<a href="https://n8n.io" target="_blank" class="logo">
-						<img :src="basePath + 'n8n-icon-small.png'" class="icon" alt="n8n.io"/>
+						<img :src="publicPath + 'n8n-icon-small.png'" class="icon" alt="n8n.io"/>
```

Comment:
can you also include a fix for the google sign in image? 
https://github.com/n8n-io/n8n/blob/5c65a73cc21ac4348eaae6b6b010980ff789cefe/packages/editor-ui/src/components/CredentialsInput.vue#L57

Reply:
Good idea. I'll do it.